### PR TITLE
[Fixes AB#685] Offers news

### PR DESCRIPTION
**Pull request by Skafec**
Added news section on offer details screen

EDIT: In case users input the wrong link, "Invalid news link!" text will be shown. (this was removed, backend is always returning URL in response)

Currently on hold until we get a new response from Backend.
@githinho tell me when backend is done 😄

### CODE REVIEW

**Comment by githinho** [comment]
The new version is live on staging.

File: src/app/offers/offer-details/offer-details.component.html [hunk 1]
```diff
@@ -182,6 +182,19 @@ <h1 class="app-basic-title">{{ states.project.name }}</h1>
                     </div>
                 </div>
             </div>
+
+            <div class="left__news">
+                <ng-container *ngIf="states.news != null">
```
Comment (mightymatth) [inline src/app/offers/offer-details/offer-details.component.html:209]: if there are no news, markup will look like this:

```
<div class="left__news">
</div>
```

We don't want that. Remove ng-container and put `*ngIf="states.news != null"` on `div.left__news`.

File: src/app/organizations/organization-details/manage-projects/manage-single-project/manage-single-project.component.scss [hunk 2]
```diff
@@ -157,6 +157,7 @@
               }
               div {
                 color: $icons-color;
+                cursor: pointer;
```
Comment (mightymatth) [inline src/app/organizations/organization-details/manage-projects/manage-single-project/manage-single-project.component.scss:160]: If something is clickable, it should be either a button or anchor link ("a" tag).

It's good to follow this: If something is clickable, and it's not a button, wrap it in anchor tag, set empty routerLink and set click listener:

```
<a [routerLink]="" (click)="openBeer($event)">
  <div>Anything clickable</div>
</a>
```

This way, this element will be clickable, visible in text readers, and have a pointer as a cursor on the desktop.

**Review by mightymatth — CHANGES_REQUESTED** [review]

**Comment by mightymatth** [comment]
If we have link previews, why don't we present the entire link preview (image, title, description)?

Consider having link like this: https://www.index.hr/vijesti/clanak/pitali-smo-struku-o-spomeniku-domovini-bukvalan-je-preskup-i-na-krivom-mjestu/2237703.aspx?index_tid=76661

Its preview (e.g. on Twitter) looks like this:

image

Meaning that image, title and description can be fetched from our link preview service.
So my idea is to make a list of items like this:

image

where each item will be clickable which will redirect the user to the given URL. If we already have these data from our link preview service, why not use them.

@filipdujmusic , can you think about this idea; do we need it now, how the items might look like, etc. I'm just mentioning it here to discuss it.

**Comment by filipdujmusic** [comment]
> If we have link previews, why don't we present the entire link preview (image, title, description)?
> 
> Consider having link like this: https://www.index.hr/vijesti/clanak/pitali-smo-struku-o-spomeniku-domovini-bukvalan-je-preskup-i-na-krivom-mjestu/2237703.aspx?index_tid=76661
> 
> Its preview (e.g. on Twitter) looks like this:
> 
> image
> 
> Meaning that image, title and description can be fetched from our link preview service.
> So my idea is to make a list of items like this:
> 
> image
> 
> where each item will be clickable which will redirect the user to the given URL. If we already have these data from our link preview service, why not use them.
> 
> @filipdujmusic , can you think about this idea; do we need it now, how the items might look like, etc. I'm just mentioning it here to discuss it.

@mightymatth why not? Your proposed layout looks good to me. I'm just trying to think one step ahead, if we are ever going to move link/file components to the right side of the offer details screen (to fill a white gap caused by large description size), would this layout fit in?

**Comment by mj-ampnet** [comment]
Yes - keep this layout that @mightymatth suggested - it used to be implemented like this.a

**Comment by mj-ampnet** [comment]
@Skafec what's happening with this feature? It would be good to have it for the launch... It shouldn't be too difficult to implement... The link preview service gives you all the data you need. If you're worried about the design, contact me on slack and we'll discuss.

**Review by mightymatth — APPROVED** [review]
It's OK for now. A new design will be made in the next pull request.

**Pull request merged**